frontend/integration-tests/tests/olm/single-installmode.scenario.ts

```diff
@@ -171,15 +163,6 @@ describe('Interacting with a `OwnNamespace` install mode Operator (Prometheus)',
     expect(crudView.successMessage.getText()).toContain('example has been updated to version');
   });
 
-  it('displays Kubernetes objects associated with the `Prometheus` in its "Resources" section', async() => {
```

Comment:
I understand that this test as written no longer applies since we're removing the row filters from the empty state, but we should try to test that the right resources are showing up in a different way. Maybe we can test that the tabs are present?

cc @alecmerdler

Reply:
We should not remove this test. If the row filters are being removed, than check that `resourceRows` are present for the associated `StatefulSets`, `Pods`, etc for the given CR.

Reply:
The row filters are only being removed if there are no results to filter - so there would not be any resourceRows, but the tests have been super flakey for me, even on master so I don't think I'll be able to dig into why there are no resourceRows being printed before I leave for PTO tomorrow

@alecmerdler @spadgett